stable/datadog/templates/NOTES.txt

```diff
@@ -64,21 +64,30 @@ agents use the same shared token.
 The datadog agent is listening on port 8126.
 {{- end }}
 
-{{- if (and (.Values.daemonset.enabled) (.Values.deployment.enabled)) }}
-
+{{- if .Values.deployment.enabled }}
 #################################################################
 ####               WARNING: Deprecation notice               ####
 #################################################################
 
-Simultaneous DaemonSet + Deployment installation within a single
-release will be deprecated in a future version. You will still be
-able to run both as separate releases with separate value files.
+* Deployment installation will be deprecated in the futur major
+  chart version (`v2.0.0`). This installation configuration is
+  triggered by the option: `deployment.enabled: true`.
+  The recommanded solution is to run the Datadog-agent with the
+  `Daemonset` configuration which is the default configuration.
+
+{{- if .Values.daemonset.enabled }}
 
-Running as a Deployment is not necessary for event collection
-anymore, thanks to the leader election feature.
+* Simultaneous DaemonSet + Deployment installation within a single
+  release will be deprecated in the futur major chart version
+  (`v2.0.0`). You will still be able to run both as separate
```

Comment:
Since the previous if case mentions that the deployment installation will be deprecated in the next major version, why does this notice add up (here and below) to a possibility of running it?

Reply:
the deprecation notice wanted to be clearer. In both usage it will be deprecated, but I wanted to distinguish the two usecase.
I can do a `if  elseif` condition if you think it will be clearer.

Reply:
I think that's fine as long as we drop the statement about running it as separate releases